hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbReaderTest.cpp

```diff
@@ -265,69 +266,57 @@ class ServiceHootApiDbReaderTest : public CppUnit::TestFixture
     CPPUNIT_ASSERT_EQUAL(0.0, node->getX());
     CPPUNIT_ASSERT_EQUAL(0.0, node->getY());
     CPPUNIT_ASSERT_EQUAL(10.0, node->getCircularError());
-    CPPUNIT_ASSERT_EQUAL(0, node->getTags().size());
-    QString tagValue = node->getTags().get(MetadataTags::HootStatus());
-    CPPUNIT_ASSERT_EQUAL(Status::Unknown1, node->getStatus().getEnum());
-    CPPUNIT_ASSERT_EQUAL(10.0, node->getCircularError());
+    CPPUNIT_ASSERT_EQUAL(1, node->getTags().size());
     CPPUNIT_ASSERT_EQUAL((long)1, node->getVersion());
     CPPUNIT_ASSERT(node->getTimestamp() != ElementData::TIMESTAMP_EMPTY);
+    HOOT_STR_EQUALS("1", node->getTags().get(MetadataTags::HootId()));
 
     node = map->getNode(2);
     CPPUNIT_ASSERT_EQUAL(Status::Unknown2, node->getStatus().getEnum());
     CPPUNIT_ASSERT_EQUAL((long)2, node->getId());
     CPPUNIT_ASSERT_EQUAL(0.1, node->getX());
     CPPUNIT_ASSERT_EQUAL(0.0, node->getY());
     CPPUNIT_ASSERT_EQUAL(11.0, node->getCircularError());
-    CPPUNIT_ASSERT_EQUAL(1, node->getTags().size());
-    tagValue = node->getTags().get("noteb");
-    CPPUNIT_ASSERT(!tagValue.isEmpty());
-    CPPUNIT_ASSERT_EQUAL(QString("n2b").toStdString(), tagValue.toStdString());
```

Comment:
Asserting that `tagValue` is not empty and then that it equals `n2b` is redundant.  Therefore I removed it and replaced it with just the last check.